src/Microsoft.TestPlatform.Common/Filtering/FilterExpression.cs

```diff
@@ -168,16 +164,16 @@ internal string[] ValidForProperties(IEnumerable<string> properties, Func<string
         /// </summary>
         internal static FilterExpression Parse(string filterString, out FastFilter fastFilter)
         {
-            ValidateArg.NotNull(filterString, "filterString");
+            ValidateArg.NotNull(filterString, nameof(filterString));
 
-            // below parsing doesn't error out on pattern (), so explicitly search for that (empty parethesis).
+            //// below parsing doesn't error out on pattern (), so explicitly search for that (empty parethesis).
```

Comment:
nit nit nit: might want to undo the changes to this comment (remove the extra `//`).